packaging/installer/kickstart-static64.sh

```diff
@@ -1,7 +1,15 @@
 #!/usr/bin/env sh
+#
 # SPDX-License-Identifier: GPL-3.0-or-later
 # shellcheck disable=SC1117,SC2039,SC2059,SC2086
-
+#
+#  Options to run
+#  --dont-wait              do not wait for input
+#  --non-interactive        do not wait for input
+#  --dont-start-it          do not start netdata after install
+#  --stable-channel         Use the stable release channel, rather than the nightly to fetch sources
+#  --local-tarball-override Use a manually provided tarball for the installation
```

Comment:
The option name is too long for no good reason. `--local-files` is good enough in my opinion.